### ISSUE TITLE
computedAsync allowSignalWrites

### ISSUE THREAD

**Issue by kkachniarz220** [nartc/ngxtension-platform#260]
Possibility to write signals in `computedAsync`
```typescript
data = computedAsync(() => {
  this.loading.set(true); // NG0600
  return this.getAdmissionTimeCallbackFn(this.from(), this.to())
    .pipe(finalize(() => this.loading.set(false)));
});
```

This will be also useful to set e.g error signal.

**Comment by RobbyRabbitman** [comment]
You can use the `untracked` function when setting signals inside computed/effect.

**Comment by eneajaho** [comment]
Hello @kkachniarz220 
When this https://github.com/nartc/ngxtension-platform/pull/257 gets released (later today), you will be able to do sth like: 

```ts
const admissionTime = computedAsync(
    () => this.getAdmissionTimeCallbackFn(this.from(), this.to()).pipe(
        map((res) => ({ status: 'loaded' as const, result: res })),
        startWith({ status: 'loading' as const, result: [] }),
        catchError((err) => of({ status: 'error' as const, error: err }))
    ),
    { requireSync: true }
);
```

This way, you don't need another signal for the loading state.

**Comment by eneajaho** [comment]
Closing this as completed. 


#257 was released in v2.0.0.

And you can also use `untracked` too. https://github.com/angular/angular/issues/53986
